frontend/src/static/js/v_segment.js

```diff
@@ -0,0 +1,16 @@
+window.vSegment = {
+  props: ['segment', 'path'],
+  template: window.$('v_segment').innerHTML,
+  data() {
+    return {
+      loaded: this.segment.authored || this.segment.lines.length < 5,
```

Comment:
I don't think we need to preload these segment with lines less than 5. Since the segment is small, loading a 5-line segment will not take much time that the user will even notice. However, as a whole, loading many of these small segments takes great time.

Reply:
The reason behind preloading segments with lines less than 5 is to display it together with the authored code. Since there is less than 5 lines of code in the un-authored segment, it is better to display together with the authored segment instead of requiring to toggle it just to see those few lines of un-authored segment. 

An example would be the following:
<img width="798" alt="Screen Shot 2019-06-11 at 11 38 14 PM" src="https://user-images.githubusercontent.com/32864116/59286363-2aabbf00-8ca2-11e9-96e2-792fbbb0bf93.png">
It will be inconvenient for the user to toggle the icon just to see line 416.

Reply:
I see. I didn't notice that. Just ignore this comment.